docs/pages/tutorial/eas/manage-app-versions.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: Manage different app versions
+sidebar_title: Manage app versions
+description: Learn about developer-facing and user-facing app versions and how to manage them automatically.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { BookOpen02Icon } from '@expo/styleguide-icons';
+
+In this chapter, we're going to configure our app to auto-increment the developer-facing app version. Learning about it will be useful before we dive into production build in the next few chapters.
+
+## Difference between developer-facing and user-facing app versions
+
+An app version is composed of two values:
+
+- The [`versionCode`](/versions/latest/config/app/#versioncode) for Android and [`buildNumber`](/versions/latest/config/app/#buildnumber) for iOS (also known as developer-facing values)
+- The [`version`](/versions/latest/config/app/#version) property in **app.json** (also known as user-facing values)
+
+## Manually manage developer-facing values
+
+Google Play Store and Apple App Store use developer-facing values to identify each unique build. Submitting builds with duplicate version numbers will result in errors, necessitating a rebuild with a new version number.
+
+To manage app versions, `android.versionCode` and `ios.buildNumber` can be set in **app.config.js** and manually updated for each new production release:
+
+```json app.config.js
+{
+  "ios": {
+    "buildNumber": "1"
+    /* @hide ... */ /* @end */
+  },
+  "android": {
+    "versionCode": "1"
+  }
+  /* @hide ... */ /* @end */
+}
+```
+
+However, we are not going to set them up manually. We are going to use EAS Build to automate this process in the next section.
+
+> **Note**: The user-facing version number is not handled by EAS. Instead, we define that in the app store developer portals before submitting our production app for review.
+
+## Automate app version management with EAS Build
+
+EAS Build can assist us in automating these values by utilizing the [remote version resource](/build-reference/app-versions/#remote-version-source). This feature automatically increments developer-facing values and keeps track of them whenever a new production release is made.
+
+In **eas.json**:
+
+- Add `cli.appVersionSource` and set it to `remote`
+- Set [`build.production.autoIncrement`](/eas/json/#autoincrement-1) to `true`:
+
+{/* prettier-ignore */}
+```json eas.json
+{
+  "cli": {
+    /* @hide ... */ /* @end */
+    /* @info Add the appVersionSource property and set it to remote*/
+    "appVersionSource": "remote"
```

Comment:
Do we want to recommend this? It makes me uncomfortable not managing it manually, it's not clear what makes it increment.